src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -178,5 +178,15 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
         }
       }
     }
+
+    "run workflows in the correct directory" in {
+      val (wma, workflowId) = runWdl(sampleWdl = SampleWdl.CurrentDirectory,
+                                     EventFilter.info(pattern = s"starting calls: whereami.whereami", occurrences = 1))
+      val outputs = wma.ask(WorkflowManagerActor.WorkflowOutputs(workflowId)).mapTo[binding.WorkflowOutputs].futureValue
```

Comment:
I have to admit that @mcovarr shamed me into doing it. I was going to leave it for a future PR